jxp/ch13ex2.pl

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use Cow;
+use Horse;
+use Sheep;
+use Mouse;
+use v5.10;
+
+say "Please enter a list of Animals";
+my $input = <STDIN>;
+
+my @animals = split " ",$input;
+
+foreach my $animal ( @animals ){
+  $animal->speak();
+}
```

Comment:
Yes, unfortunately I couldn't find a good way to do this. I tried adding ...or print ... some error message, but that didn't work and eval didn't seem to return any error message.